static/src/javascripts/projects/common/modules/experiments/contributor-email-form.js

```diff
@@ -0,0 +1,4 @@
+define([], function () {
```

Comment:
A lot less that was initially intended! I kept the file in case it needed to be there, as everything else I looked at had a companion .js file.